exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -1639,29 +1656,76 @@ func (b *Bitfinex) Subscribe(channelsToSubscribe []stream.ChannelSubscription) e
 		return err
 	}
 
-	var errs error
+	wg := sync.WaitGroup{}
+	wg.Add(len(channelsToSubscribe))
+	errC := make(chan error, len(channelsToSubscribe))
```

Comment:
This is a pattern I use a _lot_ and I'm very fond of it.
Channels are very cheap.

Considered doing it in Unsub too.
If it flies I think we can roll it out elsewhere.